pydriller/metrics/process/comm.py

```diff
@@ -0,0 +1,30 @@
+from pydriller.repository_mining import RepositoryMining
+
+class COMM():
```

Comment:
I would avoid an all uppercase class name (it doesn't follow the style of the other classes).
Anyhow, I would change the name to something else, some ideas could be:
- Metrics
- ProcessMetrics (not a big fun of this one though, not everyone is familiar with the term)

Reply:
My suggestions would be as follows:
- rename the class to "Metrics", if we expect to implement only process metrics, or
- if we want to keep it more abstract, rename the class to CommitCount (maybe extending a class Metric) and call it in a single module called metrics.py along with the future metrics.

Anyway, I leave you the last word.

Reply:
Ok. Yep we have to think it well, so that it's gonna be easy to extend in the future.
I was having a look at the process metrics:
- Commit Count
- Active Dev Count
- Distinct Dev Count 
- Normalized Lines Added 
- Normalized Lines Deleted 
- Owner’s Contributed Lines 
- Minor Contributor Count 
- Changed Code Scattering 
- Neighbor’s Active Dev Count 
- Neighbor’s Distinct Dev Count 
- Neighbor’s Commit Count 
- Neighbor’s Change Scattering 
- Owner’s Experience
- All Committer’s Experience 

What if we have this class named `ProcessMetrics`, and as methods we have these counts?
Then we will think in the future about `CodeMetrics` as well..one step at the time 😄

Reply:
Yep, definitely!